askama_parser/src/node.rs

```diff
@@ -299,7 +299,15 @@ impl<'a> Cond<'a> {
         let mut p = tuple((
             |i| s.tag_block_start(i),
             opt(Whitespace::parse),
-            ws(keyword("else")),
+            ws(alt((keyword("else"), |i| {
+                let _ = keyword("elif")(i)?;
+                Err(nom::Err::Failure(ParsingError {
+                    input: i,
+                    message: Some(Cow::Borrowed(
+                        "Unknown `elif` keyword. Did you mean `else if`?",
```

Comment:
I think we should not capitalize `Unknown` here, since this is preceded by `error: ` anyway. Also don't love the `.` here, maybe `;` or `,`?

Reply:
Let's go with `;`.